src/OrchardCore/OrchardCore.ReCaptcha.Core/ReCaptchaRenderer.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Logging;
+using OrchardCore.Localization;
+using OrchardCore.Modules;
+using OrchardCore.ReCaptcha.ActionFilters;
+using OrchardCore.ReCaptcha.ActionFilters.Detection;
+using OrchardCore.ReCaptcha.Configuration;
+using OrchardCore.ReCaptcha.TagHelpers;
+using OrchardCore.ResourceManagement;
+
+namespace OrchardCore.ReCaptcha
+{
+    internal static class ReCaptchaRenderer
```

Comment:
This should be a service that resolves the dependencies from DI. Doesn't require an interface though.

Reply:
fixed